UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/ObjectLayer.cs

```diff
@@ -98,19 +159,61 @@ public float GetObjectResistanceAt( Vector3Int position, bool isServer )
 	}
 
 
+	public bool CanEnterTileV2(Vector3Int origin, Vector3Int to, bool isServer,List<PushPull> Pushings,  GameObject context = null)
+	{
+		bool PushObjectSet = false;
+		bool CanPushObjects = false;
+
+		//Targeting windoors here
+		foreach (RegisterTile o in GetTileList(isServer).Get(to))
```

Comment:
This seems like it's the same foreach loop in 106, Can't you turn this into a function to reduce redundancy?

Reply:
well is quite small, Can do later